sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -36,6 +36,7 @@
 from __future__ import absolute_import, unicode_literals
 
 import errno
+from multiprocessing import AuthenticationError
```

Comment:
this is the wrong import :) 

should be from eventhub exceptions

Reply:
it must've auto-imported, good catch!